deployment/src/strongmind_deployment/acm.py

```diff
@@ -0,0 +1,60 @@
+import re
+from dataclasses import dataclass
+from pulumi import Output, ResourceOptions
+from pulumi_aws import acm, route53
+import pulumi
+
+
+@dataclass
+class AcmCertificateArgs:
+    stack: str
+    zone_id: str
+    tags: dict = None
+    domain: str = "prod.aws2.strongmind.com"
+
+
+class AcmCertificate(pulumi.ComponentResource):
+    def __init__(
+        self, name: str, args: AcmCertificateArgs, opts: pulumi.ResourceOptions = None
+    ):
+        super().__init__("pkg:index:AcmCertificate", name, None, opts)
```

Comment:
there's a precedence for package naming in the other components,  we should follow that.

Reply:
Agreed.

Reply:
done